x-pack/plugins/apm/server/lib/services/get_services/get_service_transaction_stats.ts

```diff
@@ -179,16 +168,18 @@ export async function getServiceTransactionStats({
           ),
         },
         transactionsPerMinute: {
```

Comment:
While you are at it: can you rename this to "throughput"?

Reply:
Above it says "avgResponseTime". Perhaps you can rename that to "averageLatency"